db/init_db.js

```diff
@@ -1,25 +1,74 @@
 // code to build and initialize DB goes here
 const {
-  client
+  client,
   // other db methods 
+  createAlbums,
+  createGenres,
+  createAlbumGenres,
+  createUser
 } = require('./index');
 
 async function buildTables() {
   try {
-    client.connect();
-
-    // drop tables in correct order
+    console.log("Building tables...");
+    await client.connect();
 
     // build tables in correct order
 
+    await client.query(`
+    DROP TABLE IF EXISTS users;
+    DROP TABLE IF EXISTS genre_albums;
+    DROP TABLE IF EXISTS genres;
+    DROP TABLE IF EXISTS products;
+
+    CREATE TABLE products (
+      id SERIAL PRIMARY KEY,
+      album_name varchar(255) UNIQUE NOT NULL,
+      artist varchar(255) NOT NULL,
+      year INT,
+      price INT DEFAULT 1999,
+      quantity INT DEFAULT 0,
+      reorder_number INT DEFAULT 0,
+      img_url varchar(255) DEFAULT 'https://cdn4.vectorstock.com/i/thumb-large/82/48/vinyl-record-blank-realistic-vinyl-disc-mockup-on-vector-17128248.jpg'
```

Comment:
Just saying I appreciate your use of DEFAULTs, I didn't realize that was a thing

Reply:
We might want to make img_url actually an array of image_urls. Currently in our JSON we have an array that looks like this: 
`"images": [
      {
        "height": 640,
        "url": "https://i.scdn.co/image/ab67616d0000b27376713b7514fdc2770ec12ab7",
        "width": 640
      },
      {
        "height": 300,
        "url": "https://i.scdn.co/image/ab67616d00001e0276713b7514fdc2770ec12ab7",
        "width": 300
      },
      {
        "height": 64,
        "url": "https://i.scdn.co/image/ab67616d0000485176713b7514fdc2770ec12ab7",
        "width": 64
      }
    ],`


If you think we should just store one image_url, ignore this and I'll change our Spotify JSON parser